WordPress/gutenberg
# Apply minimal variant to pagination dropdown

## What?
Apply `minimal` variant to pagination control in data views.

## Why?
The current implementation gives outsized importance to this element.

| Before | After |
| --- | --- |
| <img width="563" alt="Screenshot 2024-07-22 at 16 40 30" src="https://github.com/user-attachments/assets/b30a2388-7fc6-4574-b626-3b7144f562e7"> | <img width="312" alt="Screenshot 2024-08-13 at 11 22 10" src="https://github.com/user-attachments/assets/7fe5ea6d-562c-41c5-af86-967aa3cb9970"> <img width="328" alt="Screenshot 2024-08-13 at 11 22 04" src="https://github.com/user-attachments/assets/80d5dbfa-ba31-438e-8e2d-76cda2d9d8ad"> |



## Testing Instructions
1. Navigate to a data view 
2. Observe the new page selection UI
3. Check it works as before

I suspect there's a better way to handle the i18n, so please feel free to push to this branch :)
